frontend/src/tests/applicant-tests.js

```diff
@@ -12,6 +12,7 @@ export function applicantTests(api) {
     let session;
     let applicant;
     let newApplicant;
+    let resp;
```

Comment:
No global `resp` variable. It should be defined inside each function.

Reply:
Got it